### PR TITLE
Rearranged Leaderboard formatting

### PR DESCRIPTION
**Pull request by CEOofHentai**
Placed name column at the start instead of the end

### CODE REVIEW

File: src/carim_discord_bot/discord_client/arguments.py [hunk 1]
```diff
@@ -268,24 +268,24 @@ async def process_user_message_args(channel_id, parsed_args):
...
-                line_items += [r['latest_name']]
                 result_data.append(line_items)
             s = [[str(e) for e in row] for row in result_data]
             lens = [max(map(len, col)) for col in zip(*s)]
             fmt = ' '.join('{{:{}}}'.format(x) for x in lens)
             table = [fmt.format(*row) for row in s]
             formatted_result = '```\n' + '\n'.join(table) + '\n```'
             asyncio.create_task(discord_service.get_service_manager().send_message(
-                discord_service.UserResponse(channel_id, 'Leaderboard', formatted_result)
+                discord_service.UserResponse(channel_id, 'Leaderboard https://i.postimg.cc/ydFdN3s6/the-verge-icon.png', formatted_result)
```
Comment (schana) [inline src/carim_discord_bot/discord_client/arguments.py:288]: What is this?

File: src/carim_discord_bot/discord_client/arguments.py [hunk 2]
```diff
@@ -268,24 +268,24 @@ async def process_user_message_args(channel_id, parsed_args):
             for r in result.get('users', list()):
                 if stats is None:
                     stats = tuple(k for k in r.keys() if k not in ('cftools_id', 'rank', 'latest_name'))
-                    result_data.append([stat for stat in ('#',) + stats + ('name',)])
+                    result_data.append([stat for stat in ('#',) + ('name',) + stats])
                 line_items = [r['rank']]
                 for stat in stats:
+                line_items += [r['latest_name']]
```
Comment (schana) [inline src/carim_discord_bot/discord_client/arguments.py:274]: This would need to go before the for loop

**Review by schana — CHANGES_REQUESTED** [review]

File: src/carim_discord_bot/discord_client/arguments.py (hunk 1 above)
Comment (CEOofHentai) [reply src/carim_discord_bot/discord_client/arguments.py:288]: Disregard. Accidentally left in.

File: src/carim_discord_bot/discord_client/arguments.py (hunk 2 above)
Comment (CEOofHentai) [reply src/carim_discord_bot/discord_client/arguments.py:273]: Thanks. Moved.

**Review by schana — APPROVED** [review]

**Pull request merged**